pkg/types/messages.go

```diff
@@ -0,0 +1,77 @@
+package types
+
+import (
+	"time"
+
+	"github.com/nuts-foundation/nuts-auth/pkg/contract"
+	"github.com/privacybydesign/irmago"
+	"github.com/privacybydesign/irmago/server"
+)
+
+// CreateSessionRequest is used to create a contract signing session.
+type CreateSessionRequest struct {
+	// ContractType such as "BehandelaarLogin"
+	Type contract.ContractType
+	// Version of the contract such as "v1"
+	Version contract.Version
+	// Language of the contact such as "NL"
+	Language contract.Language
+	// LegalEntity denotes the organization of the user
+	LegalEntity string
```

Comment:
should this still be called LegalEntity? Or is it Organization(ID)?

Also, there's a PartyID in Nuts Core you maybe can use.